src/ducks/api-utils.test.ts

```diff
@@ -1,6 +1,6 @@
 import { promiseWithSetTimeout, stubFetch, withError, withResponse } from '../test/test-utils';
-import * as chai from 'chai';
-import * as chaiAsPromised from 'chai-as-promised';
+import chai from 'chai';
```

Comment:
skal sies at med CRA så ligger jest innlagt, som er nesten tilsvarende til chai. Men om vi vil fortsette å bruke chai så er det greit for meg :)